lib/mixpanel-node.js

```diff
@@ -19,12 +19,17 @@ var create_client = function(token, config) {
         throw new Error("The Mixpanel Client needs a Mixpanel token: `init(token)`");
     }
 
+    // Default config
     metrics.config = {
         test: false,
         debug: false,
-        verbose: false
+        verbose: false,
+        host: 'api.mixpanel.com'
     };
 
+    // Store user-provided config, overriding defaults
+    for(var i in config) { metrics.config[i] = config[i]; }
```

Comment:
Isn't this already taken care of here? https://github.com/mixpanel/mixpanel-node/blob/f60327dcbcddfaab75e7023246040ddc892e4aab/lib/mixpanel-node.js#L686-L696